perllib/FixMyStreet/App/Controller/Report/New.pm

```diff
@@ -644,6 +645,14 @@ sub setup_categories_and_bodies : Private {
             my $metas = $contact->get_metadata_for_input;
             $category_extras{$contact->category} = $metas if @$metas;
 
+            if ($c->cobrand->can('council_id')) {
```

Comment:
This makes one query per category; again, it would seem sensible to match how the priorities code handles it (which is around https://github.com/mysociety/fixmystreet/blob/afa66d3cd1b4d84e151bd8c438060fc8fc340bbe/perllib/FixMyStreet/App/Controller/Report.pm#L314 ) - fetching all the priorities and grouping them by category.

In fact, this code will not handle defect types that aren't associated with a category (which should therefore be associated with all categories), but the priority code does handle that.